src/Core/ConstraintValidator/Constraints/TypedRegex.php

```diff
@@ -34,6 +34,20 @@
  */
 class TypedRegex extends Constraint
 {
+    /**
+     * Available types
+     */
+    const TYPE_NAME = 'name';
```

Comment:
@zuk3975 Can you create an issue "Use these constants in every piece of code where it's needed" ? So we wont forget

Reply:
Maybe you could add it here #13989 ?